charts/orchestrator/values.yaml

```diff
@@ -70,86 +102,85 @@ orchestrator:
-        quarkus.log.category."io.smallrye.reactive.messaging".level = DEBUG
-        quarkus.log.category."org.kie".level = DEBUG
-        quarkus.log.category."io.quarkus.reactivemessaging".level = DEBUG
-        quarkus.log.category."io.vertx".level = DEBUG
-    spec: |+
-      flow:
-        start: PrintStartMessage
-        events:
-          - name: event1
-            source: ''
-            type: event1_event_type
-          - name: event2
-            source: ''
-            type: event2_event_type
-        functions:
-          - name: systemOut
-            type: custom
-            operation: sysout
-        timeouts:
-          eventTimeout: PT60S
-        states:
-          - name: PrintStartMessage
-            type: operation
-            actions:
-              - name: printSystemOut
-                functionRef:
-                  refName: systemOut
-                  arguments:
-                    message: "${\"event-state-timeouts: \" + $WORKFLOW.instanceId + \" has started.\"}"
-            transition: WaitForEvent1
-          - name: WaitForEvent1
-            type: event
-            onEvents:
-              - eventRefs: [ event1 ]
-                eventDataFilter:
-                  data: "${ \"The event1 was received.\" }"
-                  toStateData: "${ .exitMessage1 }"
-                actions:
-                  - name: printAfterEvent1
-                    functionRef:
-                      refName: systemOut
-                      arguments:
-                        message: "${\"event-state-timeouts: \" + $WORKFLOW.instanceId + \" executing actions for event1.\"}"
-
-            transition: WaitForEvent2
-          - name: WaitForEvent2
-            type: event
-            onEvents:
-              - eventRefs: [ event2 ]
-                eventDataFilter:
-                  data: "${ \"The event2 was received.\" }"
-                  toStateData: "${ .exitMessage2 }"
-                actions:
-                  - name: printAfterEvent2
-                    functionRef:
-                      refName: systemOut
-                      arguments:
-                        message: "${\"event-state-timeouts: \" + $WORKFLOW.instanceId + \" executing actions for event2.\"}"
-            transition: PrintExitMessage
-          - name: PrintExitMessage
-            type: operation
-            actions:
-              - name: printSystemOut
-                functionRef:
-                  refName: systemOut
-                  arguments:
-                    message: "${\"event-state-timeouts: \" + $WORKFLOW.instanceId + \" has finalized. \" + if .exitMessage1 != null then .exitMessage1 else \"The event state did not receive event1, and the timeout has overdue\" end + \" -- \" + if .exitMessage2 != null then .exitMessage2 else \"The event state did not receive event2, and the timeout has overdue\" end }"
-            end: true
\ No newline at end of file
+  sonataflows: # workflows to get deployed
```

Comment:
Do we want to use this chart to deploy GPT for workflow project scaffolding?

Reply:
@rgolangh is exploring if and how it is possible to deploy GPT which requires a dynamic plugin to exist.
https://redhat-internal.slack.com/archives/C04F2P1U0PN/p1700673649532419